### ISSUE TITLE
Issue with amp-video-docking and amp-video-iframe via Google AMP cache

### ISSUE THREAD

**Issue by carlosplayinc** [ampproject/amphtml#27559]
## What's the issue?

We are using the amp-video-iframe component with amp-video-docking.
The docked video player works as expected when viewing the original AMP page or viewing from the AMP cache via ampproject.org on iOS Safari.
The docked video player does not render when the page is viewed from google.com on iOS Safari.

## How do we reproduce the issue?

You can see the docked video player working as expected via the below urls:
1. https://www.independent.co.uk/news/uk/politics/boris-johnson-coronavirus-self-isolation-symptoms-latest-update-a9445251.html?amp
2. https://www-independent-co-uk.cdn.ampproject.org/c/s/www.independent.co.uk/news/uk/politics/boris-johnson-coronavirus-self-isolation-symptoms-latest-update-a9445251.html?amp

Steps to reproduce the docked video player not working on iOS Safari:
1. Search for 'independent uk news' via google.com search
2. Find any video-based AMP news article from the 'Latest from independent.co.uk' results
3. After viewing/skipping any pre-roll ads, trigger the docked player by interacting with the player and then scrolling the player out of view
4. The docked player should appear, but instead the video pauses as it goes out of view

## What browsers are affected?

iOS Safari version 13 (iOS 13.3.1)
Issue seen on iPhone XR.

## Which AMP version is affected?

AMP version as per console output:
Version 2004030010070

AMP components used:
https://github.com/ampproject/amphtml/tree/master/extensions/amp-video-iframe
https://github.com/ampproject/amphtml/tree/master/extensions/amp-video-docking

AMP versions used:
https://cdn.ampproject.org/v0/amp-video-iframe-0.1.js
https://cdn.ampproject.org/v0/amp-video-docking-0.1.js

**Comment by powellish** [comment]
@alanorozco any progress with this one please?

**Comment by alanorozco** [comment]
Docking is not supported in AMP Viewers in iOS, due to how they [handle scrolling and fixed elements](https://medium.com/@dvoytenko/amp-ios-scrolling-and-position-fixed-b854a5a0d451):

https://github.com/ampproject/amphtml/blob/b6313e372fdd1298928e2417dcc616b03288e051/extensions/amp-video-docking/0.1/amp-video-docking.js#L395-L402

This is due to a [Webkit bug](https://bugs.webkit.org/show_bug.cgi?id=154399) that was fixed in a recent version, though we need additional work to not use a transfer layer in runtime.

In short, this support is unfortunately blocked at the moment 😕 But I'll track support via this issue.

**Comment by powellish** [comment]
Hi Alan, has a fix happened for this yet please?

**Comment by Johrten** [comment]
Hey @alanorozco, is this still a blocked issue? 😃

**Comment by powellish** [comment]
Hi guys - any update here please? seems crazy there's no docking component for video on iOS - that's $1,000s a day lost for publishers...

**Comment by powellish** [comment]
Hey @alanorozco - is this issue still blocked?
